content/en/graphing/widgets/distribution.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Distribution Widget
+kind: documentation
+description: "Graph a metric distribution aggregated across one or several tags."
+further_reading:
+- link: "graphing/dashboards/timeboard/"
+  tag: "Documentation"
+  text: "Timeboards"
+---
+
+The Distribution visualization is another way of showing metrics aggregated across one or several tags, such as *hosts*. Unlike the [heat map][1], distribution's x-axis is the quantity rather than time.
```

Comment:
```suggestion
The Distribution visualization is another way of showing metrics aggregated across one or several tags, such as *hosts*. Unlike the [heat map][1], a distribution graph's x-axis is quantity rather than time.
```